app/views/orders/_form.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= form_with model: order do |form| %>
+
+ <% if order.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(order.errors.count, "error") %> prohibited this order from being saved:</h2>
+      <ul>
+        <% order.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.hidden_field :employee_id, value: current_employee.id %>
```

Comment:
you do not really user `:employee_id` in the controller, also you are passing `reward.id`, and `reward`, and using both to do the same thing-  find reward - I would consider to unify it. Also passing user.id as a hidden parameter might not be a good idea, since those fields are accessible from the browser.